components/page.jsx

```diff
@@ -75,13 +76,16 @@ var Page = React.createClass({
           </div>
           <Footer/>
         </div>
+
+        <ReactCSSTransitionGroup transitionName="modal">
         {this.state.modalClass
-         ? <div ref="modalHolder">
+         ? <div ref="modalHolder" key={1}>
```

Comment:
We're just giving it a static key because there will only ever be one modal in this "list". I guess it may have the side effect of having no transition effect if we were to suddenly switch from one modal to another, but that doesn't currently happen in the codebase, and even if it did, no transition might actually be the behavior we want.